src/components/language-provider.js

```diff
@@ -0,0 +1,10 @@
+import React, { memo } from 'react'
+import { IntlProvider } from 'react-intl'
+
+export const LocalizationProvider = memo(({ children, locale, messages }) => {
```

Comment:
how come the name of this file and the component it exports are named differently? should we just use a default export here?

Reply:
good point - I was originally going to export other things from this file but i'll update it for now to be default, and change the file name too